apps/wing/project.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "wing",
```

Comment:
@MarkMcCulloh If we are keeping the name of this directory as "wing" then it might just be easier to keep it as wing in this file (not sure if this is related to the current build error) though I'm okay with either

Reply:
There seems to be an issue with nx where it has trouble if the "project name" is different from the package.json name, so yeah unfortunately this will have to match the icky package name until we can get wing